src/core/theme/index.ts

```diff
@@ -58,13 +62,13 @@ const internalTokens = {
 };
 
 const importedTokens = {
-  colors,
-  spacing,
+  colors: colorTokens,
+  spacing: spacingTokens,
   typography: suomifiDesignTokens.typography,
   values: suomifiDesignTokens.values,
 };
 
-export const defaultTokens = {
+export const defaultThemeTokens = {
```

Comment:
Is this exported outside the library / part of API? If it is, then I'd try to keep it as it was.

Reply:
I don't see that it is used outside the library. Here is the two places where it is used:

---

1.) 

https://github.com/vrk-kpa/suomifi-ui-components/blob/dca31e6cc60c34523004d1c599ee5102cfba084d/src/core/theme/utils/defaults.ts#L11-L12

---

2.) 

https://github.com/vrk-kpa/suomifi-ui-components/blob/dca31e6cc60c34523004d1c599ee5102cfba084d/src/core/theme/tokens.test.tsx#L8